slides/01-introduction.qmd

```diff
@@ -141,3 +141,116 @@ inference vs. prediction
 library(tidymodels)
 ```
 
+##  {background-image="images/tm-org.png" background-size="contain"}
+
+## The whole game
```

Comment:
Can you have a slide before this that describes what we are about to show? Something like "we want you to think about 'the whole game' of modeling. That means... "